src/components/Picture/index.jsx

```diff
@@ -12,7 +12,7 @@ export default function Pictures() {
   return (
     <SPicture>
       {pictures.map((picture) => {
-        return <img src={picture.url} alt="opthalmology" />;
+        return <img src={picture.description} alt="opthalmology" />;
```

Comment:
You didn't read the message well : the src was good with {picture.url} . What needed to be change was the alt, with {picture.description}